chirp/drivers/h777.py

```diff
@@ -665,3 +689,22 @@ class BFM4Radio(H777Radio):
     def match_model(cls, filedata, filename):
         # This model is only ever matched via metadata
         return False
+
+@directory.register
+class BF1900Radio(H777Radio):
+    VENDOR = "Baofeng"
+    MODEL = "BF-1901"
+    PROGRAM_CMD = b'PWPG970'
+    ALIASES = [BaofengBF1904]
```

Comment:
We're actually trying to avoid adding new Aliases nowadays. Can you convert that into just a subclass of this and `@directory.register` it like this one? I assume the 1904 is the one with the different power levels? If so, you might also want to make that subclass expose a different set of values so it's correct?